### DISCUSSION TITLE
Problem with UDP Port Forwarding

### DISCUSSION THREAD

**Discussion by karasangry** [Openwrt-Passwall/openwrt-passwall2#285] [Q&A]
Explain how to set up. Need to create a direct access rule, but UDP port forwarding is acting weird.
If I change the global setting "UDP No Redir Ports" to "No patterns are used" - I experience problems with games and Discord. But if I change it to "All" - there are problems with video calls and DNS.
Settings Access rule:
![acl](https://github.com/xiaorouji/openwrt-passwall2/assets/138438419/6e6774c2-5517-4848-bf60-da57c84fa3e7)
Settings Shunt node:
![shunt](https://github.com/xiaorouji/openwrt-passwall2/assets/138438419/a2a90450-63c7-4b57-92e6-1f160b6aa989)
Global Forwarding Settings:
![other](https://github.com/xiaorouji/openwrt-passwall2/assets/138438419/8c15c66a-a40b-4d03-8316-a73194c53591)

**Comment by Travel2Here** [comment]
Your configuration seems to have selected direct connection for all types, so why do you still need to enable this tool? 😅
I am not sure about your specific needs, provide more details, ppl can analyze and offer assistance.

**Comment by karasangry** [comment]
Right. But this does not solve the problem with UDP ports 😞 
There is a host 192.168.1.100 (as an example) - I want all traffic from it to bypass the proxy. The settings above are what is configured now. And this does not solve the problem that UDP ports are forwarded with strange logic. I want to do the same for the guest wifi network

**Comment by Travel2Here** [comment]
I suggest that you learn about the proxy logic of Xray（by search on google), and then you will know how to meet your needs. It sounds like you want to provide proxy services only for specific devices and have other devices connect directly. This can be easily achieved, but it cannot be explained in just a few sentences. Go and learn it. Good luck!

**Comment by karasangry** [comment]
Not certainly in that way. The guest wifi network goes bypassing the proxy. The rest works through the rules.
Thank you for the direction of the search
